### ISSUE TITLE
Autopilot -> Memory param is not working

### ISSUE THREAD

**Issue by Stas-Buzunko** [Stas-Buzunko/twilio-bugs#1]
When try adding Memory to autopilot as shown in https://www.twilio.com/docs/autopilot/channels/chat#3-add-autopilot-url-to-channel Autopilot stops working. it's added to channel but it doesn't respond. cannot debug further.

specifying 2 params with & doesn't work as well

here are cases that work and that don't

twilio-chat?assistantName=Helen&TargetTask=greeting - **doesn’t work**
twilio-chat?TargetTask=greeting - **works**
twilio-chat?assistantName=Helen - **works**
twilio-chat?Memory={"CarModel":"Diablo","CarMake":"Lamborghini","CarYear":"2019"} - **doesn’t work**

ticket id 3594197
ticket was opened before 18th of Jan as on 18th i got a "template" response from an agent.
ticket was then asnwered on Mar 4, 10:43 AM PST with request to provide "more details" 
ticket was closed on Fri, 6 Mar, 12:09 PM PST

to sum up: it took support 1.5 month to request details but then it took 2 days to close my ticket.

first agent: Tom
second agent: Nico

**Comment by spalmertwilio** [comment]
Stas,  I greatly apologize for the way your request has been handled. I replied to your support request and look forward to your reply.

**Comment by Stas-Buzunko** [comment]
Response: Memory passed for inbound context in twilio chat should be encoded like 
```
twilio-chat?Memory=%7B%22CarModel%22%3A%22Diablo%22%2C%22CarMake%22%3A%22Lamborghini%22%2C%22CarYear%22%3A%222019%22%7D
```

but this doesn't answer why twilio-chat?assistantName=Helen&TargetTask=greeting - doesn’t work

**Comment by Stas-Buzunko** [comment]
“Thank you for bringing the parameter not encoded in the documentation to our notice. We will correct the documentation.”

“In the case of 
twilio-chat?assistantName=Helen&TargetTask=greeting - doesn’t work
We have found the issue, we will fix to execute the TargetTask instead”
